### PR TITLE
Feature/eui 89 search results

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-89

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] No
```

### CODE REVIEW

File: src/cases/containers/case-search/case-search.component.ts [hunk 1]
```diff
@@ -12,20 +16,90 @@ import * as fromCaseCreate from '../../store/reducers';
...
+  }
+
+  assignData(st) {
+    if (typeof st !== 'undefined' && st !== null) {
+      this.jurisdiction = st.jurisdiction.value as Jurisdiction;
+      this.caseType = st.caseType.value as CaseType;
+      this.metadataFields = st.metadataFields.value as Array<string>;
+      this.caseState = this.caseType.states[0];
+      const lfg = JSON.parse(localStorage.getItem('search-form-group-value'));
+      const search = this.searchService.search(this.jurisdiction.id, this.caseType.id, this.metadataFields, lfg, null);
```
Comment (fazanki) [inline src/cases/containers/case-search/case-search.component.ts]: I will never approve this.

File: api/lib/interceptors.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ export function successInterceptor(response) {
 }
 
 export function errorInterceptor(error) {
+    console.log(error)
```
Comment (smogese) [inline api/lib/interceptors.ts]: remove console.log

File: config/helpers.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const path = require('path');
```
Comment (smogese) [inline config/helpers.js]: What is this?

File: package.json [hunk 4]
```diff
@@ -23,6 +23,7 @@
     "lint": "npm run lint:ng && npm run lint:node",
     "test": "cd api && npm run coverage && cd .. && npm run test:ng",
     "test:ng": "ng test --watch=false --browsers=ChromeHeadless",
+    "test:ngdebug": "rimraf ./coverage && karma start",
```
Comment (smogese) [inline package.json]: Whats this for?

File: src/cases/containers/case-search/case-search.component.ts (hunk 1 above)
Comment (smogese) [reply src/cases/containers/case-search/case-search.component.ts]: Please give reasons

File: webpack.test.config.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
+const helpers = require('./config/helpers');
```
Comment (smogese) [inline webpack.test.config.js]: Why is this in here?

File: api/lib/interceptors.ts (hunk 2 above)
Comment (udaydenduluri33) [reply api/lib/interceptors.ts]: Done

File: package.json (hunk 4 above)
Comment (udaydenduluri33) [reply package.json]: Removed it.

File: config/helpers.js (hunk 3 above)
Comment (udaydenduluri33) [reply config/helpers.js]: I removed this file now

File: webpack.test.config.js (hunk 5 above)
Comment (udaydenduluri33) [reply webpack.test.config.js]: Removed it

**Pull request closed without merging**